spec/controllers/api/v1/events_controller_spec.rb

```diff
@@ -78,5 +84,12 @@
       expect(returned_json["event_name"]).to eq event4["can't be blank"]
       expect(Event.count).to eq(prev_count)
     end
+
+    it 'sends an invitation email' do
+      ActionMailer::Base.deliveries = []
+      FactoryBot.create(:event, user_id: user1.id)
+
+      expect(ActionMailer::Base.deliveries.size).to eq(0)
+    end
```

Comment:
this suggests that it...doesnt send the email?

Reply:
Yes, this is the fail scenario where the user does not fill in a required field.